pkg/apis/gke.cattle.io/v1/types.go

```diff
@@ -266,6 +271,13 @@ type GKENodePoolAutoscaling struct {
 }
 
 type GKENodeConfig struct {
+	// BootDiskKmsKey it the Customer Managed Encryption Key used to encrypt
+	// the boot disk attached to each node in the node pool. This should be
+	// of the form
+	// projects/[PROJECT_ID]/locations/[LOCATION]/keyRings/[KEY_RING_NAME]/cryptoKeys/[KEY_NAME]
```

Comment:
to align with field name :
```suggestion
	// projects/[PROJECT_ID]/locations/[LOCATION]/keyRings/[RING_NAME]/cryptoKeys/[KEY_NAME]
```

Reply:
We could use `key-ring-name` as a name for the new field to align, however, that is a personal choice

Reply:
ok, updated